app/containers/ProductView/index.js

```diff
@@ -0,0 +1,248 @@
+                    <div className="review-comment">{review.comment}</div>
+                </li>
+            ));
+        }
+        return (
+            <i className="fas fa-spinner fa-2x fa-pulse"></i>
+        );
+    }
+
+    buildMeta(meta) {
+        if (meta) {
+            return (
+                <Helmet>
+                    <title>{meta.title}</title>
+                    <meta name="description" content={meta.description} />
+                    <meta name="keywords" content={meta.keywords} />
+                </Helmet>
+            );
+        }
+        return null;
+    }
+
+    buildProduct(product) {
+        if (product) {
+            return (
+                <div className="product-panel">
+                    <div className="product-image-panel">
+                        <img className="product-image" src={product.image.large} alt={product.extra_name} />
+                    </div>
+                    <div className="product-info-panel">
+                        <div className="brand-info">
+                            <img className="brand-logo" src={product.brand.logo} alt={product.brand.name} />
+                            <div className="brand-rating">{product.review.count} reviews</div>
+                        </div>
+                        <div className="product-info">
+                            <div className="price">{product.currency.symbol}{product.price.selling}</div >
+                            <div className="brand">{product.brand.name}</div>
+                            <div className="name">{product.name}</div>
+                        </div>
+                    </div>
+                    <HashTag className="hashtags-panel" tags={product.hashtags || []} />
+                    <div className="action-panel">
+                        <div className="wishlist-btn">
+                            {/* Further update for wishlist checking */}
+                            <i className={`${true ? 'far' : 'fas'} fa-heart`} />
+                        </div>
+                        <div className="add-to-cart-button">
+                            Add To Cart
+                        </div>
+                    </div>
+                    <hr className="splitter" />
+                </div>
+            );
+        }
+        return (
+            <i className="fas fa-spinner fa-pulse"></i>
+        );
+    }
+
+    render() {
+        if (dataChecking(this.props, 'productview', 'loading')) {
+            return <h1>Loading</h1>;
+        }
+
+        const { product, reviews } = dataChecking(this.props, 'productview', 'data');
+        if (!product) {
+            return <h1>Product not found</h1>;
+        }
+
+        return (
+            <div className="container">
+                { this.buildMeta(product.meta) }
+                { this.buildProduct(product) }
+
+                <Accordion
+                    className="accordion-panel"
+                    active_icon="-"
```

Comment:
if u name it icon. can it be an icon? 

```
active_icon={() => <i class="fas fa-minus"></i>}
```

Reply:
just use 
`active_icon=<i class="fas fa-minus"></i>`